website/docs/r/log_search.html.markdown

```diff
@@ -73,7 +86,9 @@ The following arguments are supported:
 - `description` - (Optional) Description of the search.
 - `parent_id` - (Required) The identifier of the folder to create the log search in.
 - `query_string` - (Required) Log query to perform.
-- `query_parameters` - (Optional) TODO Find a good description.
+- `query_parameter` - (Block List, Optional) List of query parameters for the log search. See [query parameter schema](#schema-for-query_parameter).
```

Comment:
Calling this a "List" could be misleading. When you write out the HCL it doesn't look like a list. The example now shows how it's used, but I'd recommend adding here that this up to 10 `query_parameter` blocks can be added, one for each parameter in the `query_string`.

Reply:
made changes to the description.